netutils/libwebsockets/Makefile

```diff
@@ -0,0 +1,162 @@
+	$(LIBWEBSOCKETS_UNPACK)/lib/plat/unix/unix-fds.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/alloc.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/buflist.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/context.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/lws_dll2.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/lws_map.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/libwebsockets.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/logs.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core/vfs.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/base64-decode.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/cache-ttl/lws-cache-ttl.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/cache-ttl/heap.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/cache-ttl/file.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/dir.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/prng.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/lws-ring.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/lwsac/lwsac.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/lwsac/cached-file.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/lejp.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/misc/sha-1.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/system/system.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/system/smd/smd.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/dummy-callback.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/output.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/close.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/network.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/vhost.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/pollfd.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/service.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/sorted-usec-list.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/wsi.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/wsi-timeout.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/adopt.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/state.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/client.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/connect.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/connect2.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/connect3.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/connect4.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/sort-dns.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/core-net/client/conmon.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/pipe/ops-pipe.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/http/header.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/http/date.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/http/parsers.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/http/cookie.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/h1/ops-h1.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/h2/http2.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/h2/hpack.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/h2/ops-h2.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/ws/ops-ws.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/ws/client-ws.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/ws/client-parser-ws.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/raw-skt/ops-raw-skt.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/raw-file/ops-raw-file.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/roles/http/client/client-http.c \
+	$(LIBWEBSOCKETS_UNPACK)/lib/event-libs/poll/poll.c
+
+CSRCS += \
```

Comment:
Should this be wrapped with `if` for MBED_TLS config?

Reply:
@W-M-R please take a look at this.